sjb/config/test_cases/test_branch_origin_web_console_310.yml

```diff
@@ -21,9 +21,20 @@ extensions:
         else
           echo "IMAGE_VERSION=latest" > IMAGE_VERSION_VAR
         fi
+        # We copy oc from the image to get the right version. Use
+        # openshift/origin-cli in 3.10 and newer. Use openshift/origin for
+        # earlier releases.
+        case $PULL_BASE_REF in
+        enterprise-3.[3-9])
+          CLI_IMAGE="origin"
+          ;;
+        *)
+          CLI_IMAGE="origin-cli"
+          ;;
+        esac
```

Comment:
I'd remove the version check here and just hard-code the right value. We know this is 3.10. (Same for 3.11 below.)

Reply:
We should come back and remove some of the version sniffing from these files since it's no longer needed, but OK as a separate PR.